cli/src/main/kotlin/commands/AdvisorCommand.kt

```diff
@@ -76,12 +76,14 @@ class AdvisorCommand : CliktCommand(name = "advise", help = "Check dependencies
 
     private val globalOptionsForSubcommands by requireObject<GlobalOptions>()
 
-    private val advisorFactory by option(
+    private val providerFactory by option(
         "--advisor", "-a",
-        help = "The advisor to use, one of ${allAdvisorsByName.keys}"
+        help = "The advisor to use, one of ${allVulnerabilityProvidersByName.keys}"
     ).convert { advisorName ->
-        allAdvisorsByName[advisorName.toUpperCase()]
-            ?: throw BadParameterValue("Advisor '$advisorName' is not one of ${allAdvisorsByName.keys}")
+        allVulnerabilityProvidersByName[advisorName.toUpperCase()]
```

Comment:
There is again some confusion regarding the terms _Advisor_ and _Vulnerability provider_. Not sure, which ones should be used for the user interface. But I guess, as long as there is only a single provider, we can stick to the term _Advisor_. Only if the user can enable multiple providers, this term should be used.

Reply:
To clarify: "Advisor" is supposed to be the umbrella term; a "vulnerability provider" is a source that advises about security issues. But there are [plans](https://github.com/oss-review-toolkit/ort/issues/3317) to also advise about completely different aspects, like the quality of an Open Source project / its community health. Such feedback should still be implemented as an advisor (in my current thinking; if that does not make the data model too heterogeneous).

Reply:
Hmm technically the names here should also be changed such as the `--advisor` option, but I did not want to make this a breaking change. Should we just stick to it for now?

Reply:
@sschuberth But is this additional "Advisor" task supposed to be done by, e.g, NexusIq as well or other services. If it is the latter should they later also be called in the CLI as `advisors` ?

Reply:
> 
> 
> @oheger-bosch Hmm technically the names here should also be changed such as the `--advisor` option, but I did not want to make this a breaking change. Should we just stick to it for now?

Yes, I would say we keep the names for now.